tools/shared/src/observe-slot-text.ts

```diff
@@ -81,6 +88,12 @@ export function ObserveSlotText<T extends Constructor<ReactiveElement>>(
                 const { childNodes } = this;
                 const textNodes = [...childNodes].filter((node) => {
                     if ((node as HTMLElement).tagName) {
+                        if (exclude?.length) {
+                            const shouldExclude = exclude.some((selector) =>
+                                (node as HTMLElement).matches(selector)
+                            );
+                            return !shouldExclude;
+                        }
```

Comment:
Consider pulling this (plus a few lines above/below) into a function that could be used like `textNodes = [...childNodes].filter(notExcluded)`?

Reply:
Yeah, then it can be used in both tests. Good call!